### DISCUSSION TITLE
Welcome to libultrahdr Discussions!

### DISCUSSION THREAD

**Discussion by arifdikic** [google/libultrahdr#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gregbenz** [comment]
Just wanted to say thank you for creating and sharing this library!

**Comment by arifdikic** [comment]
You are welcome Greg! Thank you for using it.

- Arif
